pandas/tests/frame/methods/test_fillna.py

```diff
@@ -276,15 +272,13 @@ def test_fillna_dtype_conversion(self):
         expected = DataFrame("nan", index=range(3), columns=["A", "B"])
         tm.assert_frame_equal(result, expected)
 
-    @td.skip_array_manager_not_yet_implemented  # TODO(ArrayManager) object upcasting
```

Comment:
im missing something.  why does the apply_with_block version not work in the status quo?

Reply:
Was indeed not related, and those are in the meantime already removed in https://github.com/pandas-dev/pandas/pull/44571